### ISSUE TITLE
Migrate to new ESLint config file format

### ISSUE THREAD

**Issue by nickrttn** [transloadit/eslint-config-transloadit#27]
[This](https://eslint.org/docs/latest/use/configure/configuration-files-new) is going to be the default configuration file format starting in ESLint v9.0.0, which is [set to be released](https://eslint.org/blog/2023/10/flat-config-rollout-plans/) end of this year or beginning of next year.

**Comment by mifi** [comment]
can we use it with eslint 8?

**Comment by nickrttn** [comment]
Yes, it's been supported for a while now but not the default yet.

**Comment by mifi** [comment]
I just tested, but the problem is that eslint-config-airbnb  doesn't support flat config yet. So I used the [FlatCompat](https://eslint.org/docs/latest/use/configure/migration-guide#using-eslintrc-configs-in-flat-config) but as soon as I include the airbnb config, the eslint command just hangs forever.
